### PR TITLE
info file for testing of kogito-quarkus extension

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: kogito-quarkus/info.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+url: https://github.com/ppalaga/camel-quarkus # Github repo of your extension
```
Comment (geoand) [inline kogito-quarkus/info.yaml:1]: I assume you want the Kogito repo here :)

Comment (mariofusco) [reply kogito-quarkus/info.yaml:1]: @geoand sorry, excess of copy/paste :)

**Comment by maxandersen** [comment]
merged in @mariofusco - you should see stars coming in from @quarkusbot and once you have your workflow yaml in place it should start getting the events.

File: kogito-quarkus/info.yaml (hunk 1 above)
Comment (geoand) [reply kogito-quarkus/info.yaml]: No worries :)

**Pull request merged**